### PR TITLE
CP-21573

### PR DESCRIPTION
**Pull request by kostaslamda**

### CODE REVIEW

File: planex/cmd/mock.py [hunk 1]
```diff
@@ -86,13 +89,20 @@ def createrepo(pkg_dir, metadata_dir, quiet=False):
...
+        extra_conf_list):
     """
     Write a new mock config, including a loopback repository configuration
     pointing to repo_path.    Ensure that the new config file's last-modified
     time is the same as the input file's, so that the mock chroot is not
     rebuilt.
     """
+    if extra_conf_list is None:
+        extra_conf_list = []
+
```
Comment (euanh) [inline planex/cmd/mock.py]: if you define the default in the argument parser to be [] then you don't need to make this check

Comment (kostaslamda) [inline planex/cmd/mock.py]: In Python, default values are created once when the function is defined, and are reused in each successive call. This means the if the object is mutable, like a `list`, mutations will persist through different invocations. In this case, we don't mutate the object, but I think it's consider bad practice.

Comment (euanh) [inline planex/cmd/mock.py]: I'm not suggesting adding a default to the function, I'm suggesting adding it to the argument parser:
```
argparser.add_argument("--loopback-config-extra", action="append", default=[], ...)
```

Comment (kostaslamda) [inline planex/cmd/mock.py]: Aaaaaaah, sure :)

**Pull request merged**